modules/social_features/social_content_block/src/ContentBlockPluginInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\social_content_block;
+
+use Drupal\Core\Database\Query\SelectInterface;
```

Comment:
Do we need to consider making this a SelectInterface that's not database specific? Perhaps we want to use these blocks for a different data source (like search, or the API of another OS platform) in the future?

cc @ronaldtebrake

Reply:
I think that we can do it in the future when we'll entirely sure that it's really needed

Reply:
Well, it's not something that can be done in the future because it would break the interface :) But maybe it's not worth the time now.